superset-frontend/src/components/Chart/DrillDetail/DrillDetailMenuItems.tsx

```diff
@@ -26,6 +26,7 @@ import {
   extractQueryFields,
   getChartMetadataRegistry,
   QueryFormData,
+  removeHTMLTags,
   styled,
```

Comment:
Getting Following Error - export 'removeHTMLTags' (imported as 'removeHTMLTags') was not found in '@superset-ui/core'